packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -476,6 +485,34 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         );
     }
 
+    private createAnnotationDatumCopy({
+        node,
+        datum,
+    }: {
+        node: AnnotationScene;
+        datum: AnnotationProperties;
+    }): AnnotationProperties | undefined {
+        const { type } = datum;
+        if (type in annotationConfigs) {
+            const config = annotationConfigs[type];
```

Comment:
You can save some nesting here by inverting the condition:

```suggestion
        if (!(type in annotationConfigs)) {
            throw new Error(`AG Charts - ...`);
        }
        const config = annotationConfigs[type];
```